types/three/three-typedarrayutils.d.ts

```diff
@@ -0,0 +1,23 @@
+// https://github.com/mrdoob/three.js/blob/master/examples/js/TypedArrayUtils.js
+
+export namespace TypedArrayUtils {
+
+  export namespace Kdtree {
+    export class Node {
+      obj: Float32Array;
+      left: Node;
+      right: Node;
+      parent: Node;
+      depth: number;
+      pos: number;
+    }
+  }
+
+  export class Kdtree {
+    constructor(points: Float32Array, metric: any, eleSize: number);
```

Comment:
Does it make sense to type `metric` as `(a: Float32Array, b: Float32Array) => number`?

Reply:
I'm just going by the call in the JS code which is `metric( point, node.obj )` and the type of `node.obj`from this d.ts is `obj: Float32Array;`.